site/style/about-styles.css

```diff
@@ -1,32 +1,86 @@
-.sub-topic{
-   font-size: x-large;
+/* Import Google Fonts */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600;700&family=Roboto+Slab:wght@400;700&display=swap');
+
+body {
+    font-family: 'Poppins', sans-serif;
```

Comment:
Applying this font family to the entire body brings inconsistencies in nav links with other pages. Consider fixing this.

Reply:
So what do u think should be done here?

We don't interfere withe the nav link text font?

Reply:
@tmsagarofficial  I think applying the font-family to the main class only should fix this.